src/plugins/dashboard/public/url_utils/url_helper.ts

```diff
@@ -34,12 +34,14 @@ export function addEmbeddableToDashboardUrl(
   embeddableId: string,
   embeddableType: string
 ) {
-  const { url, query } = parseUrl(dashboardUrl);
+  const { url, query, fragmentIdentifier } = parseUrl(dashboardUrl, {
```

Comment:
parseUrl seems to be broken in an intermediate release and fixed in `6.13.0` https://github.com/sindresorhus/query-string/pull/222